### PR TITLE
refactor(admin-campaigns): move router to the base container

### LINKED ISSUE #220: revert(admin-campaigns): remove click preview to focus input feature
This will not be included in the MVP since we decided it was makes the code for the builder more complex than it needs to be. It might make an appearance in the future, post-MVP if we deem it valuable based on user feedback.

This is a passive issue which does not require a direct PR to resolve since it will be resolved as the project is restructured/components are split out for the MVP (as described in #210).

For reference, this was initially proposed in #108 and implemented in #145.

### LINKED ISSUE #228: refactor(admin-campaigns): move router to the top
Currently the router is buried in the builder, however, it does not serve the goals of the MVP there. We must move it to the top and figure out a different solution for the settings panels (I think `@reach/tabs`).

## Acceptance Criteria

- [x] Router is at the top.
- [x] Has routes: `/build` and `/manage` (default)
- [x] `views` directory is created with `Builder` (move all builder code here) and `Manager` (campaign list)
- [x] Implement solution for router-less settings panels (`@reach/tabs`)

### PR DESCRIPTION
**Pull request by knowler**
## Description

Resolves #228 — The router has been moved.
Resolves #220 — Focusable fields have been removed.

I think next up will be replacing Redux with Apollo/Context/a form controller (probably will go with Final Form — I experimented with it a bunch today). I plan on doing a bunch of reorganization/refactoring of a few components when I take care of this, so perhaps any improvements you notice can be do with that task.

## How Has This Been Tested?

Locally; manually.

## Types of changes

Refactoring.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: resources/js/app/admin-campaigns/views/Manager/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+export default () => (
+    <div>
+        <h1>Manager</h1>
```
Comment (ravinderk) [inline resources/js/app/admin-campaigns/views/Manager/index.jsx]: @knowler Can you start using translation functions, `__`

File: resources/js/app/admin-campaigns/views/Manager/index.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        <p>
+            <Link to="/build">Go to builder</Link>
```
Comment (ravinderk) [inline resources/js/app/admin-campaigns/views/Manager/index.jsx]: @knowler Can you start using translation functions, `__`

Comment (knowler) [reply resources/js/app/admin-campaigns/views/Manager/index.jsx]: This is temporary code so I don’t think that’s a good use of time/effort.

Comment (knowler) [reply resources/js/app/admin-campaigns/views/Manager/index.jsx]: I will use the functions for the actual implementation of the spec though. 👍

**Comment by knowler** [comment]
> My only question is that given the scope of the PR I don't see how the `SettingsPanel` component fit into this?

Good question. Since the settings panel previously used the router itself, I had to refactor it to use something else for navigation (in this case `@reach/tabs`) at the same time I just consolidated the previous side bar nav and settings panel content into the same component.

**Pull request merged**